mkosi

```diff
@@ -3793,6 +3796,8 @@ def create_parser() -> ArgumentParserMkosi:
                        help='Script to call instead of mksquashfs')
     group.add_argument("--xz", action=BooleanAction,
                        help='Compress resulting image with xz (only gpt_ext4, gpt_xfs, gpt_btrfs, gpt_squashfs, implied on tar)')  # NOQA: E501
```

Comment:
One might comment here as well, that this conflicts with `--zstd`.

Reply:
fixed